Add suggestions with fuzzy text search when no relationship is found (#2583)
PostgREST/postgrest: CHANGELOG.md

### Fixed

 - #2565, Fix bad M2M embedding on RPC - @steve-chavez

>  - #2575, Replace misleading error message by function and parameter name suggestions when no function is found - @laurenceisla
>  - #2569, Replace misleading error message by parent and child name suggestions when no relationship is found - @laurenceisla

Reviewer: ```suggestion
 - #2575, Replace misleading error message when no function is found with a hint containing functions/parameters names suggestions - @laurenceisla
 - #2569, Replace misleading error message when no relationship is found with a hint containing parent/child names suggestions - @laurenceisla
```
